src/theme/shadow/specs/default-shadow-set.ts

```diff
@@ -0,0 +1,30 @@
+import { getPrimitiveColour } from "../../colour-primitive/theme-helper";
+import { getSemanticColour } from "../../colour-semantic/theme-helper";
+import { StyledComponentProps } from "../../helpers";
+import { PrimitiveColourSet, SemanticColourSet } from "../../types";
+import { ShadowSet } from "../types";
+
+const createShadow =
+    (shadowDef: string, colourKey: string, alpha: number) =>
+    (props: StyledComponentProps) => {
+        const color =
+            getSemanticColour(colourKey as keyof SemanticColourSet)(props) ||
+            getPrimitiveColour(colourKey as keyof PrimitiveColourSet)(props);
+        return shadowDef.replace(
+            /COLOUR/g,
+            `rgba(from ${color} r g b / ${alpha * 100}%)`
+        );
+    };
+
+export const DefaultShadowSet: ShadowSet = {
+    "xs-subtle": createShadow("0 0 4px 1px COLOUR", "neutral-50", 0.24),
```

Comment:
how about appending the colour token to the length values instead of replacing it?

```
createShadow("0 0 4px 1px", "neutral-50", 0.5)
```